locale/defaultMessages.json

```
@@ -4938,6 +4938,9 @@
     "context": "tax classes card header",
     "string": "General information"
   },
+  "ThUvIL": {
+    "string": "Hidden"
```

suggestion: Let's add context for these strings, e.g.:

```suggestion
    "context": "Collection filter condition",
    "string": "Hidden"
```